src/models/darp10/external-overview.md

```diff
@@ -89,8 +90,11 @@
 |![Fn-F11](./img/fn-f11.png)|Fn+F11  |Toggle airplane mode               |
 |![Fn-F12](./img/fn-f12.png)|Fn+F12  |Suspend                            |
 |![Fn-F12](./img/fn-dia.jpg)|Fn+`    |Play/Pause                         |
-|![Fn-*](./img/fn-star.png) |Fn+*    |Toggle keyboard backlight          |
+|![Fn-*](./img/fn-star.png) |Fn+*    |Toggle keyboard backlight*         |
```

Comment:
Actually, the next three hotkeys probably all need the note for being on the 16" model only.

Also, you'll need a different marker for the 14" and 16" notes. Otherwise I can't tell which asterisk corresponds to which note. Two asterisks should suffice on one of the model.